### PR TITLE
fix: revert lifting dagre instance outside hook

### PR DESCRIPTION
**Pull request by mainawycliffe**
Caused unpredictable issues around the graph

### CODE REVIEW

**Comment by mainawycliffe** [comment]
@moshloop I have hidden the nodes during the transition period to hide the weird behavior and only show the nodes once it position, what do you think

**Comment by moshloop** [comment]
@mainawycliffe - If I look at https://deploy-preview-1889--goofy-euclid-75956c.netlify.app/catalog/e17e38f6-9a97-4c10-ae9f-459bd2f49792/relationships?outgoing=true it only shows the namespace

**Comment by mainawycliffe** [comment]
@moshloop the intermediary node for the current config wasn't being created, hence it appeared as if it only had one node, but would expand on click

**Comment by moshloop** [comment]
@mainawycliffe https://deploy-preview-1889--goofy-euclid-75956c.netlify.app/catalog/60023763-40ea-498d-ad95-6cf4c8c31e31/relationships?outgoing=true&incoming=false is only a single card?

**Comment by mainawycliffe** [comment]
@moshloop I am working on fixed for those at the moment

File: src/components/Configs/Changes/ConfigsRelatedChanges/FilterBar/ConfigRelationshipToggles.tsx [hunk 1]
```diff
@@ -28,6 +31,14 @@ export function ConfigRelationshipToggles() {
           setParams(params);
         }}
       />
+      <Toggle
+        label="Include Soft Relations"
```
Comment (moshloop) [inline src/components/Configs/Changes/ConfigsRelatedChanges/FilterBar/ConfigRelationshipToggles.tsx:35]:
```suggestion
        label="Soft"
```

File: src/ui/Graphs/RelationshipGraph.tsx [hunk 2]
```diff
@@ -71,24 +72,25 @@ export function RelationshipGraph<T extends GraphDataGenericConstraint>({
...
       if (node.data.expandable) {
-        const { x, y, width, height } = getNodesBounds([node]);
-
-        // Calculate the center of the node
-        const nodeCenterX = x + width / 2;
-        const nodeCenterY = y + height / 2;
-
-        // Move the viewport to the center of the node
-        setCenter(nodeCenterX, nodeCenterY, { duration: 800, zoom: getZoom() });
+        fitView({
```
Comment (moshloop) [inline src/ui/Graphs/RelationshipGraph.tsx:86]: We don't want to fit view - Node sizes should only be changed by the user, it should center on the node

**Pull request merged**